package.json

```diff
@@ -22,7 +22,8 @@
     "clean": "git clean -xdf",
     "postclean": "npm ci",
     "build:samples": "tsc --build --verbose tsconfig.samples.json",
-    "build": "tsc --build --verbose tsconfig.build.json",
+    "build:browser": "npm run build:browser --workspaces --if-present",
+    "build": "tsc --build --verbose tsconfig.build.json && npm run build:browser",
```

Comment:
nit: I'd run build:browser with `postbuild` instead of `&&` to have better error messages in the case of failures.